guide/onboarding/automatic-cloud-backup.md

```diff
@@ -0,0 +1,103 @@
+
+<div class="image-slide-gallery">
+
+{% include picture.html
+   image = "/assets/images/guide/onboarding/backing-up-a-recovery-phrase/automatic-cloud-backup/cloud-backup-intro.png"
+   retina = "/assets/images/guide/onboarding/backing-up-a-recovery-phrase/automatic-cloud-backup/cloud-backup-intro@2x.png"
+   class = "shadow"
+   alt-text = "Screen explaining automatic cloud backups"
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+     image = "/assets/images/guide/onboarding/backing-up-a-recovery-phrase/automatic-cloud-backup/cloud-backup-password.png"
+   retina = "/assets/images/guide/onboarding/backing-up-a-recovery-phrase/automatic-cloud-backup/cloud-backup-password@2x.png"
+   class = "shadow"
+   alt-text = "The user can set a password which will be used to encrypt their recovery phrase and act as a login"
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+     image = "/assets/images/guide/onboarding/backing-up-a-recovery-phrase/automatic-cloud-backup/cloud-backup-service.png"
+   retina = "/assets/images/guide/onboarding/backing-up-a-recovery-phrase/automatic-cloud-backup/cloud-backup-service@2x.png"
+   class = "shadow"
+   alt-text = "Screen showing saving a password to encrypt the recovery phrase"
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+     image = "/assets/images/guide/onboarding/backing-up-a-recovery-phrase/automatic-cloud-backup/cloud-backup-provider.png"
+   retina = "/assets/images/guide/onboarding/backing-up-a-recovery-phrase/automatic-cloud-backup/cloud-backup-provider@2x.png"
+   class = "shadow"
+   alt-text = "Screen of placeholder where cloud provider UI would be"
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/onboarding/backing-up-a-recovery-phrase/automatic-cloud-backup/cloud-backup-confirmation.png"
+   retina = "/assets/images/guide/onboarding/backing-up-a-recovery-phrase/automatic-cloud-backup/cloud-backup-confirmation@2x.png"
+   class = "shadow"
+   alt-text = "Screen showing completing a cloud backup"
+   width = 250
+   height = 541
+%}
+
+</div>
+
+### How to discuss/explain It
+
+One approach is to explicitly explain within onboarding carousels that a user’s recovery phrase is stored on the cloud; this means that if they lose or break their device, they can regain access to their funds by logging into their respective cloud account on a different device.
+
+On a similar note, it’s crucial that you also explain that if someone else knows their cloud account information, they can gain access to their funds, too.
+
+{% include tip/open.html color="green" icon="check" label="Do: Inform your Users" %}
+
+Before automatically backing up a recovery phrase via cloud backup, make sure the user knows what they are about to do. Once the backup is complete, provide a summary and some reassurances before moving on. This is a good place for loading interactions and animation.
```

Comment:
> This is a good place for loading interactions and animation.

What do you mean by that? Why is it a good place?

Reply:
This is because of the time delay between when users click to upload and when it completes. 

Will update 🚧